CRM/Core/Form/RecurringEntity.php

```diff
@@ -399,6 +399,7 @@ public static function postProcess($params = array(), $type, $linkedEntities = a
                 'is_active' => 1,
               );
               $excludeDateList[] = $optionGroupValue['value'];
+              // Something is wrong here. The returned BAO object is not being used.
               CRM_Core_BAO_OptionValue::create($optionGroupValue);
```

Comment:
Hmm - I'm not sure if that is wrong per-se - ie. if the option group value is not later referred to then creating it in the DB without using it seems 'ok' 

But I'm kinda confused about whether this is part of the issue or something you noticed because it seems to be on a different form?